src/block-decorations-presenter.js

```diff
@@ -0,0 +1,149 @@
+/** @babel */
+
+const {CompositeDisposable, Emitter} = require('event-kit')
+
+module.exports =
+class BlockDecorationsPresenter {
```

Comment:
Putting a link to https://github.com/atom/atom/pull/9930#discussion_r46543078 which is not getting shown anymore because some commits were introduced after the comment.